lib/ansible/plugins/doc_fragments/files.py

```diff
@@ -25,8 +25,6 @@ class ModuleDocFragment(object):
       number which will have unexpected results.
     - As of Ansible 1.8, the mode may be specified as a symbolic mode (for example, C(u+rwx) or
       C(u=rw,g=r,o=r)).
-    - As of Ansible 2.6, the mode may also be the special string C(preserve).
```

Comment:
If you do this, please add a first line item that mentions that the implicit behavior is to preserve file permissions, at this moment this is not written anywhere.

Reply:
Is that the case for _all_ modules that include this documentation? I'm not so sure about the `unarchive` module for instance, however I might be wrong.  
I'm very careful to make such assumptions - especially seeing as similar assumptions led to problems that were resolved by this PR 😄. 

If you're 100% sure that this is indeed the case for all of those modules, I'd be happy to follow up with another PR.  Otherwise I'd leave it up to the maintainers of those modules individually, as they might need more specific/detailed information.